janusgraph-test/src/test/java/org/janusgraph/core/CQLConfiguredGraphFactoryTest.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.core;
+
+import com.datastax.driver.core.Session;
+import org.apache.commons.configuration.MapConfiguration;
+import org.janusgraph.JanusGraphCassandraContainer;
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.GRAPH_NAME;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_BACKEND;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_HOSTS;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_PORT;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@Testcontainers
+public class CQLConfiguredGraphFactoryTest extends ConfiguredGraphFactoryTest {
```

Comment:
Can you move CQLConfiguredGraphFactoryTest to janusgraph-cql module,
ConfiguredGraphFactoryTest to janusgraph-backend-testutils module (you might need to introduce additional dependency),
and InmemoryConfiguredGraphFactoryTest to janusgraph-inmemory module?

This is a common practice for tests involving different backends. We have workflows like ci-backend-cql.yml which picks up tests only under certain folders.